backend/routers/projects.router.js

```diff
@@ -1,81 +1,38 @@
 const express = require("express");
 const router = express.Router();
 
-const Project = require("../models/project.model");
+const { ProjectController } = require('../controllers');
+const { verifyUser } = require('../middleware');
 
-// GET /api/projects/
-router.get("/", (req, res) => {
-  // const { headers } = req;
-  // const expectedHeader = process.env.CUSTOM_REQUEST_HEADER;
+// Display list of all Projects with GET.
+router.get('/', ProjectController.project_list);
 
-  // if (headers['x-customrequired-header'] !== expectedHeader) {
-  //     res.sendStatus(401);
-  // } else {
-  Project.find()
-    .then((projects) => {
-      if (!projects) {
-        res.json(false);
-      } else {
-        res.json(projects);
-      }
-    })
-    .catch((err) => {
-      console.log(err);
-      res.sendStatus(500).json({
-        message: `/GET Internal server error: ${err}`,
-      });
-    });
-  // };
-});
+// Create new Project with POST.
+router.post('/', ProjectController.create);
 
-router.get("/:id", (req, res) => {
-  Project.findById(req.params.id)
-    .then((project) => {
-      res.json(project);
-    })
-    .catch((err) => {
-      console.log(err);
-      res.sendStatus(500).json({
-        message: `/GET Internal server error: ${err}`,
-      });
-    });
-});
+// Display Project by id with GET.
+router.get('/:ProjectId', ProjectController.project_by_id);
 
-router.patch("/:id", (req, res) => {
-  const { headers } = req;
-  // const expectedHeader = process.env.CUSTOM_REQUEST_HEADER;
+// Update Project by id with PATCH.
+router.patch('/:ProjectId', ProjectController.update);
 
-  // if (headers['x-customrequired-header'] !== expectedHeader) {
-  //     res.sendStatus(401);
-  // } else {
-  Project.findByIdAndUpdate(req.params.id, req.body)
-    .then((edit) => res.json(req.params.id))
-    .catch((err) =>
-      res.status(500).json({
-        error: "Couldn't edit form... Try again.",
-      })
-    );
-  // };
-});
+// Delete Project by id with POST.
+router.delete('/:ProjectId', ProjectController.destroy);
 
-router.post("/", (req, res) => {
-  // const { headers } = req;
-  // const expectedHeader = process.env.CUSTOM_REQUEST_HEADER;
+// Display list of Project Event with GET.
+router.get('/:ProjectId/upcomingevents', ProjectController.event_list);
+
+// TODO: Implement Get Project members list with GET
+router.get('/:ProjectId/members', [verifyUser.isAdminByEmail], ProjectController.member_list);
+
+// TODO: Implement Add a Project members with POST
+router.post('/:ProjectId/members', [verifyUser.isAdminByEmail], ProjectController.add_member);
+
+// TODO: Implement Get a Project member with GET
+router.get('/:ProjectId/members/:MemberId', [verifyUser.isAdminByEmail], ProjectController.member_by_id);
+
+// TODO: Implement Remove a Project member with POST
+router.delete('/:ProjectId/members/:MemberId', [verifyUser.isAdminByEmail], ProjectController.remove_member);
```

Comment:
Please remove all TODOs as we discussed changes to data models that will result in API updating.